deps/uv/include/uv/win.h

```diff
@@ -517,7 +517,7 @@ typedef struct {
       /* eol conversion state */                                              \
       unsigned char previous_eol;                                             \
       /* ansi parser state */                                                 \
-      unsigned char ansi_parser_state;                                        \
+      unsigned short ansi_parser_state;                                       \
```

Comment:
Hm, doesn't this change `sizeof(uv_tty_t)` on Windows? I think this creates a one-byte gap between `ansi_csi_argc` and `ansi_csi_argv`.

Reply:
@erw7 thoughts on Ben's message ^

Reply:
The members of the `tty` union are `rd` and `wd`. Since the size of `rd` is at least 1 byte larger than `wd`, changing the size of `ansi_parser_state` in `wd` from `char` to `short` does not change the size of `uv_tty_t`. Before committing, I use `sizeof(uv_tty_t)` to make sure the size doesn't change.

Ref. https://github.com/libuv/libuv/pull/1884/files#r366560343